clients/fides-js/src/components/fides.css

```diff
@@ -195,10 +195,6 @@ div#fides-banner-inner-container {
   display: grid;
   grid-template-columns: 1fr 1fr;
   grid-column-gap: 60px;
-
-  .fides-acknowledge-button-container {
-    margin-bottom: 0px;
-  }
```

Comment:
Unrelated, but this is invalid CSS and prints a console error in the privacy center unit tests if you look closely at them: https://github.com/ethyca/fides/actions/runs/9008947049/job/24752130402#step:8:18

Reply:
speaking of which, have we considered using sass for fides.js?

Reply:
Sure! The postprocessor we use in rollup supports lots of plugins 👍

Reply:
apparently this is supposedly valid now. https://caniuse.com/?search=css%20nesting